x-pack/plugins/maps/public/classes/layers/vector_layer/vector_layer.d.ts

```diff
@@ -31,6 +33,11 @@ export interface IVectorLayer extends ILayer {
   getValidJoins(): IJoin[];
   getSource(): IVectorSource;
   getStyle(): IVectorStyle;
+  getFeatureById(id: string | number | undefined): Feature | null;
+  getPropertiesForTooltip(
```

Comment:
Only implementation in vector_layer only uses properties feature. You can remove featureId from interface